### DISCUSSION TITLE
Storing runtime flags for execution

### DISCUSSION THREAD

**Discussion by AZMCode** [nestdotland/nest.land#250] [Q&A] [answered]
As things currently stand, nest.land does not have a way to save which run flags are needed to execute a given package binary, meaning this has to be posted in a `README` and copy-pasted into the terminal. Is it within the scopes of the project to add such options in the `egg.json` file?  Something like this:
```
{
  ...,
  "flags": "--unstable -A"
}
```

**Comment by SteelAlloy** [comment, marked as answer]
Due to the nature of Deno, it is impossible to run a module with permissions defined by the author in advance, it is the user who must activate these permissions himself. 
Furthermore, as it stands, the `eggs.json` file is not published with a module. 
This would still be possible with a command like `eggs run` but I don't think it will be done because it goes against Deno's security principles 😕 

@t8  What do you think about that?

**Reply by t8** [reply]
Yes, this is true. 

Not only this, but even if this was a supported feature by Deno, we wouldn't want to transition `egg.json` to anything necessary to run a Deno project. This would be copying the infamous `package.json`, which is not ideal for Deno's ecosystem.

If flags were a supported field in configuration, the only use case I see for it would be to inform the user what permissions are needed. This really wouldn't be necessary though because someone could build a feature into `eggs` that tests for the necessary flags to run a project automatically.
